content/en/release-notes/_index.md

```diff
@@ -12,6 +12,53 @@ version: 'releases'
 title: Releases
 ---
 
+# Apache Log4j announcement
+
+Following news of [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), new patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM are in progress.
+
+Your individual CorDapps do not need to be patched as they inherit Apache Log4j from the Corda runtime.
+
+Check the list below for expected patch release dates for your version of Corda or CENM. Use the [mitigation guide](#what-you-can-do-now) to reduce your risk before upgrading to the new patch.
+
+### Apache Log4j patch release timetable
```

Comment:
This is ambiguous and should say: Corda patch release timetable for CVE-2021-44228 issue